### DISCUSSION TITLE
Support headless automatic1111 deployments

### DISCUSSION THREAD

**Discussion by cjnosal** [open-webui/open-webui#5126] [General]
**Is your feature request related to a problem? Please describe.**
1. Deploy headless automatic1111: `webui.sh --listen  --api  --api-log --nowebui`
2. Configuring AUTOMATIC1111 Base URL in AdminPanel>Images displays `Oops! The URL you provided is invalid. Please double-check and try again.`

When launched with --nowebui, automatic1111 will only respond to /sdapi requests. The automatic1111 logs shows the failing request is `HEAD / HTTP/1.1" 404 Not Found`

When redeployed without `--nowebui`, the integration succeeds and the automatic1111 logs show:
```
"GET /sdapi/v1/sd-models HTTP/1.1" 200 OK
"GET /sdapi/v1/options HTTP/1.1" 200 OK
```

**Describe the solution you'd like**
As open-webui already makes 2 /sdapi calls when viewing/updating the image configuration, the HEAD request is redundant and can be removed to support headless automatic1111 deployments. 

**Describe alternatives you've considered**
Request automatic1111 preserve the `HEAD /` endpoint when the UI is disabled.

**Comment by tjbck** [comment]
I can't seem to reproduce the issue, let me know if you manage to find the culprit and feel free to make a PR!
